### DISCUSSION TITLE
audioplayers audio_cache .play not working

### DISCUSSION THREAD

**Discussion by bhaskarvijay** [bluefireteam/audioplayers#1173] [General]
audioCatch .play was available in 0.20 version but not working in 1.0

**Comment by Adaranijo-kenneth** [comment]
found any solution yet?

**Comment by Gustl22** [comment]
AudioCache is handled by plugin, use default AudioPlayer instead:
https://github.com/bluefireteam/audioplayers/blob/main/migration_guide.md#audiocache-is-dead-long-live-sources
